tests/test_Model.py

```diff
@@ -1064,7 +1064,7 @@ def test_instance_with_multi_trip_and_release_times(mtvrptw_release_times):
            https://doi.org/10.1287/trsc.2022.1161.
     """
     m = Model.from_data(mtvrptw_release_times)
-    res = m.solve(stop=MaxIterations(20))
+    res = m.solve(stop=MaxIterations(50))
```

Comment:
I just realized that `LS::insert` is pretty suboptimal for empty solutions, because it defaults to `routes[0][0]` as a fallback if there are no neighbors. As a consequence, what always happens in the 1st search iteration is that _all_ nodes are inserted in the first route.